cue/data/gen.cue

```diff
@@ -113,6 +113,13 @@ Family: scuemata.#Family & {
                     steps: [...#Threshold]
                 } @cuetsy(targetType="interface")
 
+                // TODO docs
+                // FIXME this is extremely underspecfied; wasn't obvious which typescript types corresponded to it
```

Comment:
See:
https://github.com/grafana/grafana/blob/main/public/app/features/dashboard/state/PanelModel.ts#L148

And:
https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/transformations.ts#L32

I expect this is going to be:

```
#Transformation: [...{
    id: string
    options: {...}
}
```

Eventually the value of 'options' will have a whole registry system based on id -- there are a few more things that follow the same pattern.  Currently not extendable as plugins, but one day may be

Reply:
ahhhh, okie doke

Reply:
best to tackle in the next pass, i think